### DISCUSSION TITLE
Setting up OSD?

### DISCUSSION THREAD

**Discussion by Quads-23** [iNavFlight/inav#6569] [Q&A] [answered]
I have setup the info I want to see on my OSD, but when I look at my Video Receiver, I don't see any of the OSD, just the image. I have a runcam phoenix 2 and tx200u. I have connected the camera, gnd and 5v wires to the corresponding wires on the tx200u, and have soldered the irc tramp to vtx on the FC(Matek F722-se). The GND and 5V wires on the TX are connected to the FC as well. Is it my wiring that is keeping me from seeing the OSD on my Video RX?

**Comment by MrD-RC** [comment, marked as answer]
You need to wire the Phoenix 2 to the FC, not the TX200U. Look at the diagrams on the Matek web site.
